# microbiome cleaning
Summary of resources for identifying and removing contaminant microbial DNA from sequence-based microbiome studies.  Includes examples of studies that have transparently reported their approach.  

## Methods for identifying contaminants
1. Decontam [paper](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0605-2) [documentation](https://benjjneb.github.io/decontam/)

2. Microdecon [paper](https://onlinelibrary.wiley.com/doi/full/10.1002/edn3.11) [documentation](https://github.com/donaldtmcknight/microDecon)
3. SCRuB  [paper](https://www.nature.com/articles/s41587-023-01696-w) [documentation](https://shenhav-and-korem-labs.github.io/SCRuB/)
4. Squeegee [paper](https://www.nature.com/articles/s41467-022-34409-z) [documentation](https://gitlab.com/treangenlab/squeegee)
   

## Benchmarking contaminant identification methods
- Controlling for Contaminants in Low-Biomass 16S rRNA Gene Sequencing Experiments [paper](https://journals.asm.org/doi/10.1128/msystems.00290-19?permanently=true) [github repository](https://github.com/KarstensLab/ControllingContaminants16S)

## Studies with examples of reporting contaminant handling in their studies
- [Detection of Bacteria in Bladder Mucosa of Adult Females](https://www.auajournals.org/doi/10.1097/JU.0000000000003189)
- [Characterizing the urobiome in geriatric males with chronic indwelling urinary catheters: an exploratory longitudinal study](https://journals.asm.org/doi/10.1128/spectrum.00941-24)
- [Differences in cell-associated and cell-free microbial DNA in blood](https://www.biorxiv.org/content/10.1101/2025.02.13.638214v1) Figure 2


